compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/text/KeyMapping.desktop.kt

```diff
@@ -33,7 +33,7 @@ private val _platformDefaultKeyMapping: KeyMapping =
 internal fun createPlatformDefaultKeyMapping(platform: DesktopPlatform): KeyMapping {
     return when (platform) {
         DesktopPlatform.MacOS -> createMacosDefaultKeyMapping()
```

Comment:
Please, write a more meaningful title. It will be in the changelog

Reply:
Edited, hope this is more informative (yet succint enough)

Reply:
I removed the YouTrack number. It isn't public.

Reply:
The PR title preferable to not contain it. It is good to add it into the description